core/server/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -147,7 +147,13 @@ public FileInfo getStatusInternal(long fileId) throws AlluxioTException {
     }
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * @deprecated since version 1.1 and will be removed in version 2.0
```

Comment:
I see. That makes sense. But do you think there's only one underFS address, when given a URI?